open-metadata-implementation/repository-services/repository-services-implementation/src/main/java/org/odpi/openmetadata/repositoryservices/rest/server/OMRSRepositoryRESTServices.java

```diff
@@ -9192,7 +9193,7 @@ private String  formatNextPageURL(String    methodName,
 
                     String serverURLRoot = instance.getLocalServerURL();
 
-                    MessageFormat mf = new MessageFormat(serverURLRoot + requestURLTemplate);
```

Comment:
the fix looks good. I notice that this method is called by many placed and many/all of them supply the userId twice in the list.This does not look right to me - or am I missing something. I am thinking iff you enable this method by fixing the url then this userId issue might be what needs to be looked at next, unless it is there for a reason. What do you think @lpalashevski ?

Reply:
Resolving this discussion due to scope change #6220